rdb/src/main/java/com/instana/dc/rdb/impl/informix/InformixUtil.java

```diff
@@ -38,7 +38,10 @@ private InformixUtil() {
     public static final String TABLESPACE_USED_SQL = "SELECT(pt.npused  * pt.pagesize)  * 1024 AS used_kb,tabname FROM sysmaster:sysptnhdr pt INNER JOIN sysmaster:systabnames tn ON tn.partnum = pt.partnum where (tn.dbsname in (%s)) order by tabname desc Limit 40;";
     public static final String TABLESPACE_UTILIZATION_SQL = "select case WHEN (pt.nptotal > 0) THEN ((pt.npused) /pt.nptotal) * 100 ELSE 0 END AS table_utilization, tabname FROM sysmaster:sysptnhdr pt INNER JOIN sysmaster:systabnames tn ON tn.partnum = pt.partnum where (tn.dbsname in ('instana')) order by tabname desc Limit 40;";
     public static final String TABLESPACE_MAX_SQL = "SELECT(pt.nptotal * pt.pagesize)  * 1024 AS total_kb, tabname FROM sysmaster:sysptnhdr pt INNER JOIN sysmaster:systabnames tn ON tn.partnum = pt.partnum where (tn.dbsname in ('instana')) order by tabname desc Limit 40;";
-    //Database Queries
+
+    public static final String DB_DISK_WRITE_COUNT_SQL = "SELECT value From sysprofile where name = 'dskwrites'";
```

Comment:
Hope this you kept for supporting the SQL part as I can't see it's usage.
nitpick: either make all the query as upper case or lower case.